dojo/db_migrations/0068_file_upload.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.2.17 on 2020-12-17 22:30
+
+from django.db import migrations, models
+import dojo.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dojo', '0067_max_dupes'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FileUpload',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=100, unique=True)),
+                ('file', models.FileField(upload_to=dojo.models.UniqueUploadNameProvider('uploaded_files'))),
+            ],
+        ),
+        migrations.AddField(
+            model_name='engagement',
+            name='files',
+            field=models.ManyToManyField(blank=True, editable=False, to='dojo.FileUpload'),
```

Comment:
Does this have to be a many-to-many? I think each file can only ever belong to at most 1 engagement/finding/test?

Reply:
That would be the hope, but could be nasty to debug that if a lot of files are present and you forgot where you put it the first time.